antora-ui-camel/src/css/catalog.css

```
@@ -0,0 +1,26 @@
+.catalog-container {
+  margin-top: 20px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.catalog-entry {
+  width: 250px;
+  min-height: 140px;
+  padding: 15px;
+  margin: 5px;
+  border-style: solid;
+  border-color: #eee;
+  border-width: 2px;
```

Use variables, `--color-smoke-90` is near, not sure about `2px` (borders are just about the only place where `px` makes sense):

```suggestion
  border: 2px solid var(--color-smoke-90);
```